package.json

```
@@ -181,18 +192,27 @@
     "rollup-plugin-terser": "^7.0.2",
     "shx": "^0.3.4",
     "tslib": "^2.3.1",
-    "typescript": "^4.5.5"
+    "typescript": "^4.5.5",
+    "vite": "*"
   },
   "resolutions": {
     "date-fns": "2.27.0"
   },
   "peerDependencies": {
     "@babel/helper-module-imports": ">=7.12",
     "@babel/types": ">=7.13",
+    "aslemammad-vite-plugin-macro": "*",
```

```suggestion
    "aslemammad-vite-plugin-macro": ">=1.0.0-alpha.1",
```
Or, you can consider releasing v1 at this point.

---

let's keep it as is, and see what the real typed-macro do!